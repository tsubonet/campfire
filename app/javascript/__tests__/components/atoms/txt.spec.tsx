import * as React from 'react'
import Txt from '../../../components/atoms/txt'
import * as renderer from 'react-test-renderer'

test('render test', () => {
  const component = renderer.create(<Txt>Test</Txt>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
