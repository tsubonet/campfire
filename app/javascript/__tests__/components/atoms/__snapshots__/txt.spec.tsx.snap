// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<div
  className="sc-bdVaJa cnIEpz"
>
  Test
</div>
`;
